opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -995,8 +995,17 @@ def record_exception(
                 type(exception), value=exception, tb=exception.__traceback__
             )
         )
+        module = (
+            exception.__module__
+            if exception.__class__.__module__ != "builtins"
```

Comment:
you can drop `__class__` here too

Reply:
If I drop it fails for builtins. Think we can use `type(exception).__module__`